README.md

```diff
@@ -143,6 +142,22 @@ Optionally, use the following steps to import the project to Eclipse:
     Two mock users are defined for local development:
     - User: `user`, password: `user` to browse books
     - User: `admin`, password: `admin` to manage books and orders
+    
+## Using IntelliJ Idea (Community and Ultimate)
+
+IntelliJ can handle the project more or less out-of-the-box. Since some of the event handlers in the project rely on
+the code generated from the CDS model the build path of the project (module) needs to be extended
+with the folder containing the generated code. In order to add the generated code you need to add the 'gen' folder
```

Comment:
Interesting. IntelliJ does not seem to be able to auto-detect this from the dynamic compile source directory that is added by the `cds-maven-plugin` to the Maven build. Eclipse (m2e) and VS Code (which uses m2e as well) seem to be able to detect this automatically.

Reply:
I had to add it manually. Since IntelliJ is quite popular those days I think it's worth an addition here.